internal/domain/pull_request.go

```diff
@@ -0,0 +1,30 @@
+package domain
+
+type PullRequestEvent struct {
```

Comment:
Consider using a more complex type (like `uint` for IDs) to reflect that these fields should never hold negative values. This helps to mitigate logical errors since IDs should always be non-negative.